websites/A/Anix/metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "819696394180100106",
+		"name": ".vahya"
+	},
+	"service": "Anix",
+	"description": {
+		"en": "Anix offers a freemium anime streaming experience with premium features, allowing you to watch anime online for free without registration."
+	},
+	"url": [
+		"anix.vc",
+		"anix.to"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/gtqJrml.png",
```

Comment:
[Logos](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#logo) must be 512x512px